app/views/dashboard/settings/app-setup.blade.php

```diff
@@ -52,8 +52,10 @@
                                     <div class='form-group'>
                                         <label>Banner Image</label>
                                         @if($banner = Setting::get('app_banner'))
-                                        <div class='well'>
+                                        <div id="banner-view" class='well'>
                                             <img src='data:{{ Setting::get("app_banner_type") }};base64,{{ $banner }}' style='max-width: 100%' />
+                                            <br /><br />
```

Comment:
There isn't a proper class yet, which means there is no decent margin at the bottom. We'll sort that soon.